.github/workflows/test.yaml

```diff
@@ -70,8 +70,7 @@ jobs:
           otp-version: ${{ env.otp }}
           elixir-version: ${{ env.elixir }}
       - name: Start epmd
-        run: cmd /c "START /b epmd"
-        working-directory: ${{ env.INSTALL_DIR_FOR_OTP }}/erts-13.0/bin
+        run: cmd /c "START /b /d %RUNNER_TEMP%\.setup-beam\otp\erts-13.0\bin epmd"
```

Comment:
I made this breaking change in setup-beam, sorry about that. :(

Reply:
No worries, the path breaks whenever we bump ERTS anyway :smile:

Reply:
something like this should work:

```yaml
- run: $(elixir -e 'IO.puts "#{:code.root_dir()}/erts-#{:erlang.system_info(:version)}"')/bin/epmd.exe
  shell: bash
```

Reply:
yeah that should do it:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/76071/190382296-ab9ff57c-2018-46ff-b170-0ab3c7b002c6.png">

(https://github.com/wojtekmach/public_actions/actions/runs/3059706522/jobs/4937395180)

Reply:
Beautiful, will do that!

(fyi I think `wojtekmach/public_actions` is private :smile:)

Reply:
@wojtekmach works great, thanks!! FTR it also needs `-deamon` at the end :)

Reply:
I am 95% sure the -daemon flag does not work on Windows.

Reply:
Without it the CI would block, and worked fine after adding it, so that's curious.